HOWTO.md

```diff
@@ -0,0 +1,115 @@
+# How To use 2DECOMP&FFT
+Full informations on how to use the 2DECOMP&FFT library together with some related theory are available [here](https://2decomp-fft.github.io/).
+
+The 2D Pencil Decomposition API is defined in three Fortran modules which should be used by applications as:
```

Comment:
is defined with three Fortran modules which can be used by applications as:

Reply:
Ok modified